README.md

```diff
@@ -1 +1,40 @@
-# spec
+# OpenEvents
+
+Events are everywhere.  However, event publishers tend to describe events differently.
+
+The lack of a common way of describing events means developers must constantly re-learn how to receive events.  This also limits the potential for libraries, tooling and infrastructure to aide the delivery of event data across environments, like SDKs, event routers or tracing systems.  The portability and productivity we can achieve from event data is hindered overall.
+
+Enter OpenEvents, a specification for describing event data in a common way.  OpenEvents seeks to ease event declaration and delivery across services, platforms and beyond.
+
+OpenEvents is a new effort and it's still under active development.  However, its working group has received a surprising amount of industry interest, ranging from major cloud providers to popular SaaS companies.  Our end goal is to offer this specification to the [Cloud Native Computing Foundation](https://www.cncf.io/).
+
+## Roadmap
+
+*January*
+
+* Establish governance, contributing guidelines and initial stakeholders.
+* Draft educational materials on how to become a stakeholder.
+* Draft educational materials on use-cases.
+* Iterate on version 0.1 of the specification.
+
+*February*
+
+* Iterate on version 0.1 of the specification.
```

Comment:
redundant. Last bullet about finalization assumes this.

Reply:
:thumbsup: